src/foam/nanos/controller/ApplicationController.js

```diff
@@ -129,37 +145,42 @@ foam.CLASS({
   methods: [
     function init() {
       this.SUPER();
-      var self = this;
 
-      this.getCurrentUser();
+      var self = this;
+      self.clientPromise.then(function() {
+        self.getCurrentUser();
```

Comment:
I kindof think you shoudl set the subSubContext here rather than in the postset you did earlier, I think it's more clear why you're setting it if you do this, and any race conditions between creating child element and the setting of subSubContext will be more apparent.